### PR TITLE
Output gradients in export_testcase

### PR DESCRIPTION
**Pull request by ir5**
I implemented new option in `export_testcase` to output gradients as well. This may be useful for the development of chainer-compiler.

### CODE REVIEW

**Comment by ir5** [comment]
I noticed that the name of gradient files should be consistent with the names in .onnx file, which is based on `str(id(param))`. I'll change this code.

**Comment by ir5** [comment]
After #118 is merged, `str(id(param))` part should be reverted to `namedparams`.

**Comment by shinh** [comment]
LGTM, but how about adding a simple test?

**Comment by ir5** [comment]
Sure, I'll try to write a simple test.

**Comment by ir5** [comment]
Add some test for `export_testcase`. The test just checks the existence of generated .pb files.

File: tests/test_export_testcase.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            L.Linear(None, 100),
+            F.relu,
+            L.Linear(100, 10)
+        )
+        x = np.zeros((1, 3, 28, 28), dtype=np.float32)
+        self.x = chainer.as_variable(x)
+
+    def test_export_testcase(self):
+        # Just check the existence of pb files
+        path = 'out/test_export_testcase'
```
Comment (disktnk) [inline tests/test_export_testcase.py]: This code works on Windows, but it's better to use `os.path.join` and `"out"` is got from `onnx_chainer.testing.test_onnxruntime.TEST_OUT_DIR`. However related to this directory will be fixed in #117 and next PR , so it's fine like this.
